lang/en/campaigns.php

```diff
@@ -468,4 +468,19 @@
         'public'    => 'Public campaign',
         'review'    => 'Awaiting Review',
     ],
+    'exports'                      => [
+        'created_by'      => 'Created by',
+        'size'            => 'Size',
+        'type'            => 'Type',
+        'status'          => 'Status',
+        'status_finished' => 'Finished',
+        'status_scheduled'=> 'Scheduled',
+        'status_failed'   => 'Failed',
+        'status_running'  => 'Running',
+        'type_assets'     => 'Assets',
+        'type_entities'   => 'Entities',
+        'created'         => 'Created',
```

Comment:
Created is already translated in `campaigns.invites.fields.created`, please don't add more work for our poor translators :)